src/main/resources/com/google/api/codegen/php/discovery_fragment.snip

```diff
@@ -126,27 +126,45 @@
       @end
 
     @end
-  @end
-  @if context.hasRequestField(method)
+    @if context.hasRequestField(method)
+
+      ${@requestBodyName()} = new {@context.getRequestClassName(method)}($client);
+      @if context.isPatch(method)
+        // {@TODO()} Assign values to desired properties of the `{@requestBodyName()}` object to be changed.
+      @else
+        // {@TODO()} Assign values to desired properties of the `{@requestBodyName()}` object.
+      @end
+
 
-    ${@requestBodyName()} = new {@context.getRequestClassName(method)}($client);
-    @if context.isPatch(method)
-      // {@TODO()} Assign values to desired properties of the `{@requestBodyName()}` object to be changed.
-    @else
-      // {@TODO()} Assign values to desired properties of the `{@requestBodyName()}` object.
     @end
 
+    @if context.hasMediaDownload(method)
+      ${@optParamsName()} = array();
+      @if context.hasMediaUpload(method)
```

Comment:
Media upload normally passes additional keyword parameters to the method, which is done in a labeled array in PHP. If there is already page-streaming or media download that refers to a provided `optParams` variable, then the user will need to add media upload parameters to this variable; otherwise, the user can just add them to the method signature as an array literal if desired (which is how it appears in the docs).